kevinene91/Store-Manager-API
# #161378879 database setup

# What does this PR do?
It has the Database setup of v2 of the app 

# Description of Task Completed?
Data should be saved in a Postgres Database 

# How to Manually test this 
-Install [Postgresql](https://www.postgresql.org/download/) in your Computer 
-Create a database  postgres command `CREATE DATABASE storemanage;`
- Clone the repository `git clone https://github.com/kevinene91/Store-Manager-API.git`
- Change into the app directory `cd Store-Manager-API`
- get this branch code `git fetch origin ch-database-161378879`
- Navigate the folder structure
- create a virtual environment `python3 -m pyvenv venv`
- pip install -r requirements.txt
Export environment variables
     - export SECRET_KEY ="random string here"
    - export APP_SETTINGS="development"
     - export DATABASE_URL="dbname=storemanager user= password= host=localhost"
-  run `python run.py` 
- Check the tables created via the following commands
- `postgres psql` accesing postgres database
- `\c storemanager`  accesing the storemanager database
- `dt` lists all tables created

### Relevant PVT story? 

[161378879](https://www.pivotaltracker.com/n/projects/2203367/stories/161378879)
